meta-lmp-base/recipes-bsp/u-boot/u-boot-base-scr/uEnv.txt.in

```diff
@@ -2,3 +2,4 @@ bootcmd_dtb=load ${devtype} ${devnum}:1 ${fdt_addr_r} ${fdtfile}
 bootcmd_load_k=load ${devtype} ${devnum}:1 ${kernel_addr_r} ${kernel_image}
 bootcmd_run=@@KERNEL_BOOTCMD@@ ${kernel_addr_r} - ${fdt_addr_r}
 bootcmd=run bootcmd_dtb; run bootcmd_load_k; run bootcmd_run
+daiane=1
```

Comment:
looks like this debug line is not need anymore.

Reply:
I though I have removed it, thanks for noticing

Reply:
Please squash the commit `9f09fd007cfd11801f773faa0c015a6a500237c8` into this one.

`git rebase -i` should help you to do this.